roles/openshift_service_catalog/files/kubeservicecatalog_roles_bindings.yml

```diff
@@ -0,0 +1,214 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: service-catalog
+objects:
+
+- kind: ClusterRoleBinding
+  apiVersion: v1
+  metadata:
+    name: template-broker-caller-binding
+  roleRef:
+    name: system:openshift:templateservicebroker-client
+  groupNames:
+  - system:unauthenticated
+  - system:authenticated
+  - system:anonymous
```

Comment:
you definitely do NOT want this role.  in general i think you're working off an old version of my template.  see the updated PR for the final iteration of the roles:
https://github.com/openshift/origin/pull/14630